tests/integration/metrics_test.go

```diff
@@ -211,3 +213,46 @@ func TestMetricsHealth(t *testing.T) {
 		t.Fatalf("expected '0' from etcd_server_health_failures, got %q", hv)
 	}
 }
+
+func TestMetricsRangeDurationSeconds(t *testing.T) {
+	integration.BeforeTest(t)
+	clus := integration.NewCluster(t, &integration.ClusterConfig{Size: 1})
+	defer clus.Terminate(t)
+
+	client := clus.RandClient()
+
+	_, err := client.Put(context.Background(), "my-namespace/foobar", "data")
+	if err != nil {
+		t.Fatal(err)
+	}
+	_, err = client.Put(context.Background(), "my-namespace/foobar1", "data")
+	if err != nil {
+		t.Fatal(err)
+	}
+	_, err = client.Put(context.Background(), "namespace/foobar1", "data")
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
- Can we use a loop to populate any data?
- Please use require.NoError(t, err) to simplify the error check. (this comment applies to the following multiple places as well)